src/pages/en/user/email/verification.html

```diff
@@ -1,5 +1,5 @@
 ---
-subject: "${brand} verification code: ${code}"
+subject: "${code} is your ${brand} verification code"
```

Comment:
This initial version where code was at the end of the sentence was by design in that way. On the phones in the notifications beginning of the message is shown, but not the end.  
Moving code to the beginning of the message might reveal it in the push notification when device is locked and unwanted person has access to locked phone.

@raphaelrobert please comment if this is still the possible threat or not anymore.

CC  @sinawire

Reply:
Good catch @jenskasemets. It's still valid that we should only have the code in the body of the email, not the subject.

Reply:
@raphaelrobert should this be rolled back or should it be changed fully to remove code from the subject line?

Reply:
The whole idea was to move it to the beginning of the subject line so one can see the code in the push notification. This decreases the drop off rate. In our scenario one has to log in to their email account, open the email, ah wait, there is another interesting email .... and now the user is distracted and is not verifying its account.

I understand the security aspect. I think our users are well aware of security and therefore have push notifications hidden while phone is locked. I see a lot of services doing it that way we proposed - even my bank. Attached an example of github also doing it.

Yes, Wire is security but since 2018 Wire is also about making revenue. I think we gain more from adding the code at the beginning of the subject line than having it only in the body. Who is the final decision maker on that?

![Github verification code](https://user-images.githubusercontent.com/37803118/98341894-e005b200-200f-11eb-91cb-c96e62823a03.jpeg)